modules/fundamental/src/sbom/model/details.rs

```diff
@@ -147,10 +148,12 @@ impl SbomAdvisory {
     }
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize, ToSchema)]
+#[derive(Clone, Debug, Serialize, Deserialize, ToSchema, SimpleObject)]
+#[graphql(concrete(name = "SbomStatus", params()))]
 pub struct SbomStatus {
     pub vulnerability_id: String,
     pub status: String,
+    #[graphql(skip)]
```

Comment:
Seems like this is where our DB entity model differing (for good reasons) from the API DTO model is causing some friction?

Reply:
@bobmcwhirter, I suppose so but it doesn't matter if structures are living in the entity or the DTO modules, we just need to declare them as Graphql objects, and dig down the nested structures.